src/Widgets/RichLabel.vala

```diff
@@ -57,7 +57,7 @@ public class Tootle.Widgets.RichLabel : Label {
 
         if ("@" in url || "tags" in url) {
             var query = Soup.URI.encode (url, null);
-            var msg_url = "%s/api/v1/search?q=%s&resolve=true".printf (accounts.formal.instance, query);
+            var msg_url = "%s/api/v2/search?q=%s&resolve=true".printf (accounts.formal.instance, query);
```

Comment:
This looks fine, though it will make application incompatible with older versions of API. What about making a fallback to v1 when v2 is unavailable? I think you can make workaround for now, and then this will be decoupled to separate API class

Reply:
Yes, I'm going to implement it, What do you think if tootle could get information about instance https://mastodon.example/api/v1/instance , this request can help us implement this behavior.